docs/anonymity-roadmap.rst

```diff
@@ -0,0 +1,51 @@
+﻿.. -*- coding: utf-8-with-signature; fill-column: 77 -*-
+
+==============================
+Anonymity Development Roadmap
+==============================
+
+
+
+Development phases
```

Comment:
i think we can remove this section... especially since most of the dev work is done and this section hasn't been updated in a long time.